benchmark_imagenet100.md

```diff
@@ -0,0 +1,8 @@
+# Benchmark results
+
+On ImageNet100 , we train base columnformer config for 100 epochs : 
+
+| commit | # of params | list_width | Imagenet100 |
+|--------|-------------|------------|-------------|
+| 690dd2c733f1fa029e43e87cf5ce99a5c3b9060 8| 47M |  [8, 12, 16]  |      22.8%       |
+
```

Comment:
This is great. Can we rename this file to `RESULTS.md`?

A few suggested changes:

```suggestion
# Results

## ImageNet-100

Columnformer performance trained on [ImageNet-100](https://huggingface.co/datasets/clane9/imagenet-100) at 128x128 resolution for 100 epochs.

| description | commit | # of params | geometry | accuracy |
|------------|--------|-------------|-----------|----------|
| baseline | [690dd2c](https://github.com/IhabBendidi/columnformers/tree/690dd2c733f1fa029e43e87cf5ce99a5c3b90608)| 47M |  [8, 12, 16]  |      22.8%       |

```